zerver/openapi/zulip.yaml

```diff
@@ -11504,6 +11528,11 @@ paths:
                                 custom profile field in user-facing
                                 settings UI for configuring custom
                                 profile fields for this account.
+                            display_in_profile_popover:
+                              type: boolean
+                              description: |
+                                If true this external accounts link will display
+                                in user profile popover.
```

Comment:
Let's leave this out from `realm_default_external_accounts`. I don't think it'll make sense for these server-level defaults to ever specify that a field should display in the profile summary; it should be something the realm admins choose when they enable the field.

That's especially so given that we plan to limit the number of fields in the profile summary to something like two of them.

Reply:
done.